web/frontend/package.json

```diff
@@ -55,7 +55,7 @@
     "@viamrobotics/eslint-config": "^0.2.6",
     "@viamrobotics/prettier-config": "^0.3.4",
     "@viamrobotics/prime": "0.5.9",
-    "@viamrobotics/prime-blocks": "^0.0.24",
+    "@viamrobotics/prime-blocks": "file:../../../prime/packages/blocks",
```

Comment:
This should not be checked in - we want to reference an actual `@viamrobotics/prime-blocks` version, not your local relative path. If your PRIME PR is already merged, then you'll just need:

```
npm install -D @viamrobotics/prime-blocks@latest
```